.gitattributes

```diff
@@ -0,0 +1,9 @@
+# This file is documented at https://git-scm.com/docs/gitattributes.
+#
+# Linguist-specific attributes are documented at
+# https://github.com/github/linguist.
+
+# Collapse generated code in code reviews.
+**/zz_generated.*.go linguist-generated=true
+api/config/v1alpha1/zz_generated.*.go linguist-generated=true
+api/config/v1beta1/zz_generated.*.go linguist-generated=true
```

Comment:
1. Good to know that GitHub obeys `linguist-generated=true`; GitHub doesn't obey `binary`.  On the flip side, there are lots of local tools that know about `binary` but don't know about `linguist-generated=true`. It would probably be good to include both.

2. Why is this 3 lines? Doesn't the first line cover the latter 2?